accepted/2020/mono-convergence/globalization-support-in-.net5.md

```diff
@@ -0,0 +1,90 @@
+
+## .NET Core ICU dependencies
+All of the existing ICU dependencies in .NET Core today are located inside System.Private.CoreLib. The .NET Core does not install ICU and relies on the unmanaged implementation to be available on the system.
+
+The existing implementation requires ICU version is [50](https://github.com/dotnet/runtime/blob/4480bdbe66c55caceb64fc3477010435ba70c4ec/src/libraries/Native/Unix/System.Globalization.Native/pal_icushim.c#L60) or higher.
+
+The implementation files which reference globalization interop are:
+
+* [System.Private.CoreLib/src/System/Globalization/IdnMapping.Icu.cs](https://github.com/dotnet/runtime/tree/87f391001eca716a8db896f5c3855d33fe30aca8/src/libraries/System.Private.CoreLib/src/System/Globalization/dnMapping.Icu.cs)
+* [System.Private.CoreLib/src/System/Globalization/CalendarData.Icu.cs](https://github.com/dotnet/runtime/tree/87f391001eca716a8db896f5c3855d33fe30aca8/src/libraries/System.Private.CoreLib/src/System/Globalization/CalendarData.Icu.cs)
+* [System.Private.CoreLib/src/System/Globalization/CultureData.Icu.cs](https://github.com/dotnet/runtime/tree/87f391001eca716a8db896f5c3855d33fe30aca8/src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.Icu.cs)
+* [System.Private.CoreLib/src/System/Globalization/TextInfo.Icu.cs](https://github.com/dotnet/runtime/tree/87f391001eca716a8db896f5c3855d33fe30aca8/src/libraries/System.Private.CoreLib/src/System/Globalization/TextInfo.Icu.cs)
+* [System.Private.CoreLib/src/System/Globalization/CompareInfo.Icu.cs](https://github.com/dotnet/runtime/tree/87f391001eca716a8db896f5c3855d33fe30aca8/src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.Icu.cs)
+* [System.Private.CoreLib/src/System/Globalization/JapaneseCalendar.Icu.cs](https://github.com/dotnet/runtime/tree/87f391001eca716a8db896f5c3855d33fe30aca8/src/libraries/System.Private.CoreLib/src/System/Globalization/JapaneseCalendar.Icu.cs)
+* [https://github.com/dotnet/runtime/blob/88cc71b26ba1eb430aaeeebddcd5675b3009f2ee/src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs#L127](https://github.com/dotnet/runtime/blob/88cc71b26ba1eb430aaeeebddcd5675b3009f2ee/src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs#L127)
+
+Existing test coverage checks the implementation which is available. It does not have any expected list of data included therefore does not check whether same set of data is available everywhere.
+
+## .NET Globalization Invariant Mode
+There is already support for the globalization [invariant mode](https://github.com/dotnet/runtime/blob/master/docs/design/features/globalization-invariant-mode.md) since .NET Core 2.0. This will be the most size effective globalization mode for .NET5 which can be set by the developers. We are going to use existing logic and tweak it where necessary to allow IL Linker to remove any globalization code from the final app when running in invariant mode.
```

Comment:
```suggestion
There is already support for the globalization [invariant mode](https://github.com/dotnet/runtime/blob/master/docs/design/features/globalization-invariant-mode.md) since .NET Core 2.0. This will be the most size effective globalization mode for .NET 5 which can be set by the developers. We are going to use existing logic and tweak it where necessary to allow IL Linker to remove any globalization code from the final app when running in invariant mode.
```